web/locales/en/messages.ftl

```diff
@@ -109,6 +109,7 @@ cookies = Cookies
 faq = FAQ
 content-license-text = Content available under a <licenseLink>Creative Commons license</licenseLink>
 share-title = Help us find others to donate their voice!
+email-subscription-title = SIGN UP FOR EMAIL UPDATES
```

Comment:
can we have this in normal casing and then do CSS text-transform to uppercase it.